pkg/exchange/okex/exchange.go

```diff
@@ -339,3 +339,30 @@ func (e *Exchange) QueryKLines(ctx context.Context, symbol string, interval type
 	return klines, nil
 
 }
+
+func (e *Exchange) QueryOrder(ctx context.Context, q types.OrderQuery) (*types.Order, error) {
+	if len(q.Symbol) == 0 {
```

Comment:
The purpose of `Exchange` is encapsulating the underlying details of okex. User will not know what `InstrumentID` is. User only knows what a `Symbol` is, so it would be better not to use `InstrumentID` in the error message.

Reply:
Updated.